pkgs/applications/networking/mailreaders/mailspring/default.nix

```diff
@@ -0,0 +1,92 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+let mailspring-deb = pkgs.stdenv.mkDerivation rec {
+  name = "mailspring-deb";
+  version = "1.6.3";
```

Comment:
newest Version is 1.7.8 with hash 1kh5q5jhbjyfn9rxinwsnfm6v6dmjh8xrra8p2jii83d7f0vyzr0. May wanna update this